mysql-test/suite/innodb/t/parallel_dblwr_encrypt.test

```diff
@@ -0,0 +1,92 @@
+# Test Parallel double write buffer encryption
+
+--source include/have_innodb.inc
+--source include/not_embedded.inc
+--source include/have_debug.inc
+
+SET GLOBAL innodb_page_cleaner_disabled_debug=ON;
+--let $MYSQLD_DATADIR=`SELECT @@datadir`
+
+CREATE TABLE t1(a TEXT) ENCRYPTION='Y';
+
+INSERT INTO t1 (a) VALUES ('Abracadabra is of unknown origin, and its first occurrence is');
+INSERT INTO t1 (a) VALUES ('Abracadabra is of unknown origin, and its first occurrence is');
+INSERT INTO t1 (a) VALUES ('Abracadabra is of unknown origin, and its first occurrence is');
+INSERT INTO t1 (a) VALUES ('Abracadabra is of unknown origin, and its first occurrence is');
+INSERT INTO t1 (a) VALUES ('Abracadabra is of unknown origin, and its first occurrence is');
+INSERT INTO t1 (a) VALUES ('Abracadabra is of unknown origin, and its first occurrence is');
+
+CREATE TABLE t2(a TEXT) ENCRYPTION='Y';
+
+INSERT INTO t2 (a) VALUES ('Abracadabra is of unknown origin, and its first occurrence is');
+INSERT INTO t2 (a) VALUES ('Abracadabra is of unknown origin, and its first occurrence is');
+INSERT INTO t2 (a) VALUES ('Abracadabra is of unknown origin, and its first occurrence is');
+INSERT INTO t2 (a) VALUES ('Abracadabra is of unknown origin, and its first occurrence is');
+INSERT INTO t2 (a) VALUES ('Abracadabra is of unknown origin, and its first occurrence is');
+INSERT INTO t2 (a) VALUES ('Abracadabra is of unknown origin, and its first occurrence is');
+
+CREATE TABLE t3(a TEXT) ENCRYPTION='Y';
+
+INSERT INTO t3 (a) VALUES ('Abracadabra is of unknown origin, and its first occurrence is');
+INSERT INTO t3 (a) VALUES ('Abracadabra is of unknown origin, and its first occurrence is');
+INSERT INTO t3 (a) VALUES ('Abracadabra is of unknown origin, and its first occurrence is');
+INSERT INTO t3 (a) VALUES ('Abracadabra is of unknown origin, and its first occurrence is');
+INSERT INTO t3 (a) VALUES ('Abracadabra is of unknown origin, and its first occurrence is');
+INSERT INTO t3 (a) VALUES ('Abracadabra is of unknown origin, and its first occurrence is');
+
+SET GLOBAL innodb_page_cleaner_disabled_debug=OFF;
+SET GLOBAL innodb_buf_flush_list_now=ON;
+SET GLOBAL innodb_fast_shutdown=2;
```

Comment:
Why not just kill_mysqld.inc?

Reply:
because we would like to have all dirty pages flushed and then verify the doublewrite buffer.
A normal shutdown removes parallel doublewrite file. So fast_shutdown=2 is used to retain parallel doublewrite buffer file and verify its contents.

Reply:
trying with kill_mysqld.inc

Reply:
Done